dts/bindings/serial/aesc,uart.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2025 aesc silicon
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: aesc UART - This binding gives a base representation of the aesc UART
```

Comment:
Please use the newly introduced `title` key as well as `description` and simplify the text accordingly.